projects/core/src/lib/users/user-session/user-session.component.scss

```diff
@@ -1,9 +1,32 @@
-dl {
-    margin: 0;
-    dt {
+.pair {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: start;
```

Comment:
From the compiler:
> WARNING: autoprefixer: /home/beilschmidt/git/geoengine-ui/projects/core/src/lib/users/user-session/user-session.component.scss:5:3: start value has mixed support, consider using flex-start instead

Reply:
[caf80ff](https://github.com/geo-engine/geoengine-ui/pull/585/commits/caf80ff70acc4e70cccb1ceffa616ed062bfe61e)